frontend/src/deadlineList/components/userAvatar.jsx

```diff
@@ -13,10 +13,20 @@ const UserAvatar = ({ width, src, ...props }) => {
 			case 3:
 				translateX = 'transform -translate-x-3/2'
 		}
+		let border = 0
+		if (width <= 250) {
+			border = 0
+		} else if (width <= 300) {
+			border = 2
+		} else if (width <= 400) {
+			border = 4
+		} else {
```

Comment:
no need for this else statement.. just put the default in border in the let border value up